src/main/java/com/mybookfinder/enterprise/service/BookServiceStub.java

```diff
@@ -13,7 +13,7 @@ public BookServiceStub(){
     public BookServiceStub(IBookDAO bookDAO){
 
     }
-    public Book save(Book book){
+    public Book save(Book book) throws Exception {
```

Comment:
Added "throws Exception" to save functions in order to catch potential exceptions. For example: What if the database being saved to is offline?
Source: Module 5 video TDDBDD Unit Test Example With Mockito and IntelliJ Idea